include/engine/routing_algorithms/routing_base_mld.hpp

```diff
@@ -30,13 +30,14 @@ namespace
 // Unrestricted search (Args is const PhantomNodes &):
 //   * use partition.GetQueryLevel to find the node query level based on source and target phantoms
 //   * allow to traverse all cells
-inline LevelID getNodeQueryLevel(const partition::MultiLevelPartitionView &partition,
-                                 NodeID node,
-                                 const PhantomNodes &phantom_nodes)
+inline LevelID
+getNodeQueryLevel(const datafacade::ContiguousInternalMemoryDataFacade<Algorithm> &facade,
```

Comment:
Leave the previous formatting and replace ContiguousInternalMemoryDataFacade with AlgorithmDataFacade